apps/kaizen/models.py

```
@@ -28,30 +24,35 @@ class SubTask(BaseModel):
-        (5, "Very Good"),
+        (4, "Happy"),
+        (5, "Very Happy"),
+    ]
+
+    PRODUCTIVITY_CHOICES = [
+        (1, "Very Unproductive"),
+        (2, "Unproductive"),
+        (3, "Neutral"),
+        (4, "Productive"),
+        (5, "Very Productive"),
     ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)